### DISCUSSION TITLE
Experimental buckets to allow prototyping

### DISCUSSION THREAD

**Discussion by erkist** [cdfoundation/sig-events#68] [Ideas]
Would it be feasible to have experimental buckets to hold new events, or new versions of events, that hasn't passed through the whole discussion and review process?

If used responsibly, it could enable us to do more prototyping and experiments with events, but still allow the standard to grow in a controlled way.

Experimental buckets could either be for buckets we may want to add to the standard (e.g. "continuous-verification-experimental") or related to already existing buckets, but where we want some updates to be made (e.g. "source-code-version-control-experimental").

We should have some more lightweight review process for experimental buckets as well, and we should make very clear that things may change before it is officially made part of the spec, but the concept could allow us to "test drive" stuff a bit more before we include it in the standard, IMHO.

**Comment by m-linner-ericsson** [comment]
For creating new events and experimenting should we call it something in line with CloudEvents and call it extensions ( https://github.com/cloudevents/spec/blob/v1.0.1/primer.md#cloudevent-attribute-extensions)?
